new 2.3.1 cffi preparation breaks on arm

2.3.1 contains @scauligi 's merged cffi updates. On both Fedora and Debian, our new release is failing to build on at leastr AArch64 and arm7hl. Here's some example output:

```c
build/temp.linux-aarch64-3.9/_notcurses.c: In function ‘_cffi_f_ncplane_styles_set’:
build/temp.linux-aarch64-3.9/_notcurses.c:25052:3: warning: ‘ncplane_styles_set’ is deprecated [-Wdeprecated-declarations]
25052 |   { ncplane_styles_set(x0, x1); }
      |   ^
In file included from ../include/notcurses/direct.h:4,
                 from build/temp.linux-aarch64-3.9/_notcurses.c:570:
../include/notcurses/notcurses.h:3966:10: note: declared here
 3966 | API void ncplane_styles_set(struct ncplane* n, unsigned stylebits)
      |          ^~~~~~~~~~~~~~~~~~
build/temp.linux-aarch64-3.9/_notcurses.c: In function ‘_cffi_d_ncplane_vprintf’:
build/temp.linux-aarch64-3.9/_notcurses.c:25425:34: error: incompatible type for argument 3 of ‘ncplane_vprintf’
25425 |   return ncplane_vprintf(x0, x1, x2);
      |                                  ^~
      |                                  |
      |                                  int
In file included from ../include/notcurses/direct.h:4,
                 from build/temp.linux-aarch64-3.9/_notcurses.c:570:
../include/notcurses/notcurses.h:1734:64: note: expected ‘va_list’ but argument is of type ‘int’
 1734 | ncplane_vprintf(struct ncplane* n, const char* format, va_list ap){
      |                                                        ~~~~~~~~^~
build/temp.linux-aarch64-3.9/_notcurses.c: In function ‘_cffi_f_ncplane_vprintf’:
build/temp.linux-aarch64-3.9/_notcurses.c:25469:38: error: incompatible type for argument 3 of ‘ncplane_vprintf’
25469 |   { result = ncplane_vprintf(x0, x1, x2); }
      |                                      ^~
      |                                      |
      |                                      int
In file included from ../include/notcurses/direct.h:4,
                 from build/temp.linux-aarch64-3.9/_notcurses.c:570:
../include/notcurses/notcurses.h:1734:64: note: expected ‘va_list’ but argument is of type ‘int’
 1734 | ncplane_vprintf(struct ncplane* n, const char* format, va_list ap){
      |                                                        ~~~~~~~~^~
build/temp.linux-aarch64-3.9/_notcurses.c: In function ‘_cffi_d_ncplane_vprintf_aligned’:
build/temp.linux-aarch64-3.9/_notcurses.c:25484:50: error: incompatible type for argument 5 of ‘ncplane_vprintf_aligned’
25484 |   return ncplane_vprintf_aligned(x0, x1, x2, x3, x4);
      |                                                  ^~
      |                                                  |
      |                                                  int
```

You can see a buildlog at:

https://kojipkgs.fedoraproject.org//work/tasks/5902/68215902/build.log
https://buildd.debian.org/status/package.php?p=notcurses&suite=experimental#problem-1

It looks like the same problem on both distros, affecting the same architectures. My first thought is that we might just be missing an `#include <stdarg.h>`, causing `va_list` to be interpreted as `int`. @scauligi , is it possible for you to look at this?

--- a/cffi/src/notcurses/build_notcurses.py
+++ b/cffi/src/notcurses/build_notcurses.py
@@ -43,7 +43,7 @@ lines = re.sub(r'(?m)(^(ALLOC |__attribute__ .*)?static (.*[^{]\n)+.*)\{$(?s:.*?
 
 # fix missing struct definitions
 # it doesn't actually matter what these are typedef'd to
-lines = 'typedef int sigset_t;\ntypedef int va_list;' + lines
+lines = 'typedef ... sigset_t;\ntypedef ... va_list;\n' + lines
 
 # remove various compiler/preprocessor hints that cffi doesn't understand
 lines = re.sub(r'\b(API|RESTRICT|ALLOC|static) ', '', lines)

